stock_tracking_prodlot/i18n/de.po

```diff
@@ -0,0 +1,28 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* stock_tracking_prodlot
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-02-01 15:07+0000\n"
+"PO-Revision-Date: 2015-02-01 16:08+0100\n"
+"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.5.4\n"
+"Language: de\n"
+
+#. module: stock_tracking_prodlot
+#: model:ir.model,name:stock_tracking_prodlot.model_stock_production_lot
+msgid "Serial Number"
+msgstr "Seriennummer"
+
+#. module: stock_tracking_prodlot
+#: model:ir.model.fields,field_description:stock_tracking_prodlot.field_stock_production_lot_tracking_id
+#: field:stock.production.lot,tracking_id:0
+msgid "pack"
+msgstr "Packung"
```

Comment:
Yes, german is a bit more complex than other languages in thes respect. As a pack in this context is a box, carton, or container or the like. So this should be a noun as commented already. I would stick with capitalization here...